### PR TITLE
Add banned message

### PR DESCRIPTION
**Pull request by Marcono1234**
Recently the first few reports have been created by players whose Minecraft account has been banned.
Therefore it might be good to add a message for this, pointing the users to the relevant help page.

I have labelled this with `discussion needed` because this mostly affects MCPE, so it would be good to get some feedback from the helpers and moderators active on that project.

### CODE REVIEW

**Review by urielsalis — APPROVED** [review]

File: assets/messages.yml [hunk 1]
```diff
@@ -209,6 +209,23 @@ messages:
...
+        - realms
+        - web
+      name: Banned
+      shortcut: ban
+      message: |-
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        This is an account issue. We do not have the tools to help you with this issue on this bug tracker.
+        Additionally bans are permanent and non-negotiable. Please have a look at the help article "[Why have I been banned from Minecraft?|https://help.minecraft.net/hc/en-us/articles/360052618531-Why-have-I-been-banned-from-Minecraft-]" and the [Community Standards|https://www.minecraft.net/en-us/community-standards].
```
Comment (SPGoding) [inline assets/messages.yml:227]: Perhaps add a comma after "Additionally"?

Comment (RickTheGeezer) [reply assets/messages.yml:227]: Definitely needs a comma there. Otherwise looks fine.

File: assets/messages.yml [hunk 2]
```diff
@@ -209,6 +209,23 @@ messages:
...
+        - mcpe
+        - realms
+        - web
+      name: Banned
+      shortcut: banned
+      message: |-
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        This is an account. We do not have the tools to help you with this issue on this bug tracker.
```
Comment (Jack-McKalling) [inline assets/messages.yml]: "This is an account issue. (...)". Without referring to an account _issue_, it'll read weird. The "this" refers to the ticket, and a ticket is not an account.

File: assets/messages.yml (hunk 1 above)
Comment (MikeUnverferth) [reply assets/messages.yml:227]: Do we really need to have it say "Additionally bans are permanent and non-negotiable." on the bug tracker? Directing them to the help article should be enough. It's not really the bug tracker's job to convey that information. Also, what happens if Microsoft ever changes the policy to add an appeal process? Then all those comments are now incorrect.

Comment (MikeUnverferth) [reply assets/messages.yml]: I edited out the statement about bans being permanent.

**Review by SPGoding — APPROVED** [review]

**Review by RickTheGeezer — APPROVED** [review]

**Pull request merged**